core/components/molecules/fileUploader/FileUploaderList.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import FileUploaderItem, { FileItem } from './FileUploaderItem';
+import { BaseProps, extractBaseProps } from '@/utils/types';
+
+export interface FileUploaderListProps extends BaseProps {
+  /**
+   * <pre style="font-family: monospace; font-size: 13px; background: #f8f8f8">
+   * List of files
```

Comment:
Please write a clear description if list is a fileList type or an array of file object, also not clear if the file is a native file type or a user created object only. User may conflict this with the native file types please mention if it expects native file types or not. Ref https://developer.mozilla.org/en-US/docs/Web/API/FileList